homeassistant/components/control4/light.py

```diff
@@ -47,14 +47,16 @@ async def async_setup_entry(
     async def async_update_data_non_dimmer():
         """Fetch data from Control4 director for non-dimmer lights."""
         try:
-            return await director_update_data(hass, entry, CONTROL4_NON_DIMMER_VAR)
+            return await update_variables_for_entity(
+                hass, entry, {CONTROL4_NON_DIMMER_VAR}
```

Comment:
According to typing - this should not be a passing a set.
I think it's not picked up by mypy because `async_update_data_non_dimmer` is untyped.

Reply:
changed type to set